.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on:
+  - push
+  - pull_request
+
+jobs:
+  Quality:
+    strategy:
+      # Don't fast-fail on tag and master
+      fail-fast: ${{ github.event_name == 'pull_request' || (github.ref !=
+        'refs/heads/master' && !startsWith(github.ref, 'refs/tags/')) }}
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11-alpine
+        env:
+          POSTGRES_DB: test_egapro
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_USER: postgres
+        options: >-
```

Comment:
Do we really need those options ? The less config, the better :)

Reply:
Removed